maven-settings.xml

```diff
@@ -0,0 +1,46 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                       https://maven.apache.org/xsd/settings-1.0.0.xsd">
+ <profiles>
+    <profile>
+      <id>release</id>
+
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+
+      <repositories>
+        <repository>
+          <id>confluent-other</id>
+          <name>Confluent</name>
+          <url>https://jenkins-confluent-packages-beta-maven.s3-us-west-2.amazonaws.com/6.2.0-beta210310224144-cp8/1/maven</url>
+        </repository>
+
+        <!-- needed for HTML API doc generation post-release only --> 
+        <repository>
+          <id>ksqldb-staging</id>
+          <name>ksqlDB staging</name>
+          <url>https://ksqldb-maven.s3.amazonaws.com/maven/</url>
+        </repository>
+      </repositories>
+
+      <pluginRepositories>
+
+        <pluginRepository>
+          <id>confluent-other</id>
+          <name>Confluent Plugin Repository</name>
+          <url>https://jenkins-confluent-packages-beta-maven.s3-us-west-2.amazonaws.com/6.2.0-beta210310224144-cp8/1/maven</url>
```

Comment:
We need to update https://github.com/confluentinc/ksql/blob/0.17.0-ksqldb/mkdocs.yml#L252 to be `6.2.0-beta210310224144-cp8`. Can you slip this change into this PR?

Reply:
done @vcrfxia. Thanks for catching that - how did you know to check that? And is the script not currently set up to update to the latest hotfix beta tag? Aka is this something I should've updated and therefore should document somewhere for next time 😄

Reply:
It's under the "Docs branch cut" section of the release checklist (in wiki). Specifically, there's an item that says "mkdocs.yml versions updated (specifically, ksqldbversion , kstreamsbetatag, and kstreamsbetabuild)"